giskard_vision/__init__.py

```diff
@@ -1,5 +1,5 @@
+from giskard_vision.landmark_detection import detectors
```

Comment:
I think now that we copied and created our own scan API, we should also register the detectors only when the scan is imported, so you can remove this import here, and introduce: https://github.com/Giskard-AI/giskard/blob/af8dcbd402c5c18f656a07b2f16debe5d637244d/giskard/scanner/__init__.py#L12-L23 in `giskard_vision/scanner/__init__.py`